packages/flutter_tools/lib/src/commands/build.dart

```diff
@@ -88,11 +88,11 @@ abstract class BuildSubCommand extends FlutterCommand {
       );
     } else {
       globals.printStatus(
-        'Building without sound null safety',
+        'Building without sound null safety ⚠️',
```

Comment:
You'll need to add this emoji to https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/base/logger.dart#L717-L726

So that we don't try to print it on old windows terminals

Reply:
Thanks, done in https://github.com/flutter/flutter/pull/110998/commits/24991eff2cb87dad3146f9385ac8ca5cf44e3fcc